recipes/thrust/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+from conans import ConanFile, tools
+import os
+
+class ThrustConan(ConanFile):
+    name = "thrust"
+    license = "Apache License 2.0"
+    description = ("Thrust is a parallel algorithms library which resembles"
+                   "the C++ Standard Template Library (STL). Thrust's high-level "
+                   "interface greatly enhances programmer productivity while "
+                   "enabling performance portability between GPUs and multicore CPUs."
+                   "Interoperability with established technologies "
+                   "(such as CUDA, TBB, and OpenMP) facilitates "
+                   "integration with existing software.")
+    topics = ("conan", "thrust", "parallel", "header_only")
+    homepage = "https://thrust.github.io/"
+    url = "https://github.com/conan-io/conan-center-index"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+    
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", src=self._source_subfolder, dst="licenses")
+        self.copy("*", src=self._source_subfolder+"/thrust", dst="include/thrust")
```

Comment:
```suggestion
        self.copy("*.h", src=os.path.join(self._source_subfolder, "thrust"), dst=os.path.join("include", "thrust"))
```

Reply:
I came back to copy everything as there are subdirs and thrust uses different types of files in addition to  .h files.